docs-web/configuration/transportserver-resource.md

```diff
@@ -366,6 +367,41 @@ secure-app   46sm
 
 In the kubectl get and similar commands, you can also use the short name `ts` instead of `transportserver`.
 
+### Using Snippets
+
+Snippets allow you to insert raw NGINX config into different contexts of NGINX configuration. In the example below, we use snippets to configure [access control](http://nginx.org/en/docs/stream/ngx_stream_access_module.html) in a TransportServer:
+
+```yaml
+apiVersion: k8s.nginx.org/v1alpha1
+kind: TransportServer
+metadata:
+  name: cafe
+spec:
+  host: cafe.example.com
+  snippets: |
```

Comment:
the field needs to be updated

Reply:
@LorcanMcVeigh there is another omission in the docs - we need to have snippets added to the table in this section https://docs.nginx.com/nginx-ingress-controller/configuration/transportserver-resource/#transportserver-specification Could you add that?

Reply:
I've added the section `using-snippets` that describes them and how they're used. It's unnecessary to add it twice. Is there something missing or is there a benefit to adding it in twice?

Reply:
the tables describe all the supported fields of the resource. so this is for completeness and consistency